iam-login-service/src/test/java/it/infn/mw/iam/test/notification/RegistrationFlowNotificationTests.java

```diff
@@ -328,4 +345,73 @@ public void testRejectFlowMotivationNotifications() throws Exception {
 
   }
 
+  @Test
```

Comment:
If I put this tests in a new file, tests are not run by GitHub and test coverage is showing 40%

Reply:
> If I put this tests in a new file, tests are not run by GitHub and test coverage is showing 40%

The name of the new file ended with 'Tests'? If not, it is ignored